app/src/main/java/no/usn/mob3000/ui/screens/info/docs/DocumentationScreen.kt

```diff
@@ -93,7 +92,8 @@ fun DocumentationScreen(
         floatingActionButton = {
             FloatingActionButton(
                 onClick = onCreateDocumentationClick,
-                containerColor = DefaultButton
+                containerColor = MaterialTheme.colorScheme.primary,
+                modifier = Modifier.shadow(15.dp)
```

Comment:
[See this comment](https://github.com/frigvid/mob3000-group11/pull/19/files#r1840963289).